dev/partkey.md

```diff
@@ -0,0 +1,165 @@
+---
+numbersections: true
+title: "Algorand Key Specification"
+date: \today
+abstract: >
+    This document specifies the list of keys and their capabilities in Algorand.
+---
+
+Overview
+========
+
+An algorand node uses two kind of cryptographic keys:
```

Comment:
Should we explain these are keys for participation?


From `goal` perspective, when we create a participation, my understanding is that both the keys below are part of the "participation key".

In the code https://github.com/algorand/go-algorand/blob/5d55a675688b92348a43c6b49ee1bf76faa41d85/data/account/account.go#L156, what is called "participation keys" below is called "voting keys", and what is called "selection credential" below is called "VRF key".

I strongly prefer the code naming as opposed to the naming below.

Would you mind changing to the code naming?

Reply:
If it's ok with you, I'd like to change the `participation keys` -> `voting keys` and the call the `selection credential` -> `VRF selection key`. The reason is that we interchangeably use these, and I'd like to try and clear this out.